tests/Microsoft.Identity.Web.Test.Common/TestConstants.cs

```diff
@@ -81,5 +81,16 @@ public static class TestConstants
         public const string ConfidentialClientKeyVaultUri = "https://buildautomation.vault.azure.net/secrets/AzureADIdentityDivisionTestAgentSecret/";
         public const string ConfidentialClientId = "16dab2ba-145d-4b1b-8569-bf4b9aed4dc8";
         public const string ConfidentialClientLabTenant = "72f988bf-86f1-41af-91ab-2d7cd011db47";
+
+        //This value is only for testing purposes. It is for a certificate that is not used for anything other than running tests
+        public const string certificateX5c = @"MIIDHzCCAgegAwIBAgIQM6NFYNBJ9rdOiK+C91ZzFDANBgkqhkiG9w0BAQsFADAgMR4wHAYDVQQDExVBQ1MyQ2xpZW50Q2VydGlmaWNhdGUwHhcNMTIwNTIyMj
```

Comment:
I got this from [MSAL](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/aef655d4575d5c4849de9f8a1df0ff7ac38affe6/tests/Microsoft.Identity.Test.Common/TestConstants.cs#L117). @jennyf19 let me know if there's a better way to get cert data or this is fine.

Reply:
yes...this should be great.

---
In reply to: [406834661](https://github.com/AzureAD/microsoft-identity-web/pull/93#discussion_r406834661) [](ancestors = 406834661)